supervisor/snapshots/snapshot.py

```diff
@@ -370,10 +374,11 @@ def _folder_save(name):
             try:
                 _LOGGER.info("Snapshot folder %s", name)
                 with SecureTarFile(tar_name, "w", key=self._key) as tar_file:
-                    tar_file.add(
+                    recursively_add_directory_contents_to_tar_file(
```

Comment:
Make the same also for the add-ons data folder

Reply:
I still don't get what folder you mean. An addon is snapshotted in `addon.snapshot`.
Please give me a hint which line you mean so I can apply the changes you are requesting.

Reply:
https://github.com/home-assistant/supervisor/blob/dev/supervisor/addons/addon.py#L574